docs/releasing.md

```
@@ -0,0 +1,158 @@
+
+Source and tests folders must contain just source files and README.mds, nothing else. Make sure that all `__pycache__` and `.mypy_cache` directories are deleted (the latest are hidden on Unit systems!)
+
+### Build the distribution archive
+
+```bash
+make distribution-archives
+```
+
+This command should finish with message:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)